src/components/basicComponents/ShowSuggestions.vue

```diff
@@ -1,13 +1,24 @@
 <template>
   <div>
     <IconSuggestions />
-    <p>
-      <span>dynamic number here</span>
+    <p v-if="Object.keys(feedbacks).length !== 1">
```

Comment:
Object.keys(feedbacks).length - you can put it to const to not repeat 3 times

Reply:
I created feedbacksLength props and put the word Suggestions into span, the code looks now: 
<p>
      <span>{{ feedbacksLength }}</span>
      <span> Suggestion</span>
      <span v-if="feedbacksLength !== 1"> Suggestions</span>
    </p>
Is that solution ok?